content/en/docs/instrumentation/js/serverless.md

```diff
@@ -173,3 +173,120 @@ function.
 
 You should now be able to view traces produced by OpenTelemetry from your Lambda
 function in the backend!
+
+## GCP function
+
+This example shows how to instrument [http triggered function](https://cloud.google.com/functions/docs/writing/write-http-functions) using GCP UI.
+
+### Creating function
+
+Login to GCP selecting the project where your function should be placed. From serverless select Cloud Functions. Select create function and select environment. This example has been implemented using second generation. 
+
+### Setup environment variable for otelwrapper 
+
+Add following runtime environment variable NODE_OPTIONS and provide value 
+
+```shell
+--require ./otelwrapper.js
+```
+
+### Select runtime
+This demonstration used node.js 16.
```

Comment:
Does this also work with other Node.JS versions? If so maybe add a short sentence about that.

Reply:
So far I have only tested using version 16